docs/documentation/patrol-vs-code-extension.mdx

```diff
@@ -0,0 +1,135 @@
+# Guide for Patrol VS Code extension
+
+## How to setup
+
+1. Install the extension from VS Code Marketplace. It requires Dart extension - 
```

Comment:
Do we already have a beta version in Marketplace? We could link it here, probably the redirect would open IDE with extension page